packages/sfui/frameworks/react/components/SfFormControl/SfFormHelperText.tsx

```diff
@@ -0,0 +1,14 @@
+import { PropsWithChildren } from 'react';
```

Comment:
This component is missing from vue implementation

Reply:
I didn't include it in the PoC, because it's very small and would look exactly the as in React. Although, I can added it if you want.

Reply:
no needed, if you think it will be easy to do the same in vue :)